plugins/poll/db/migrate/20230614041219_delete_duplicate_poll_votes.rb

```diff
@@ -3,12 +3,20 @@
 class DeleteDuplicatePollVotes < ActiveRecord::Migration[7.0]
   def up
     execute <<~SQL
-      DELETE FROM poll_votes
-      WHERE (poll_id, user_id, updated_at) NOT IN (
-        SELECT poll_id, user_id, MAX(updated_at)
-        FROM poll_votes
-        GROUP BY poll_id, user_id
+    DELETE FROM poll_votes
```

Comment:
Just to confirm: this keeps only the _first_ vote/option voted of a user in regular polls.

This sounds correct to me, maybe I would keep the last one? But I think either option is fine.

Reply:
I updated the test so that it is more clear. The migration keeps the more recently created vote. 👍